BoxContentSDK/BoxContentSDK/Requests/BOXCollaborationCreateRequest.m

```diff
@@ -8,12 +8,19 @@
 #import "BOXAPIJSONOperation.h"
 #import "BOXCollaboration.h"
 
+@interface BOXCollaborationCreateRequest()
+
+@property (nonatomic, readwrite, strong) BOXAPIItemType *itemType;
```

Comment:
Personally I like to have a properties instead of manipulating iVars directly. But yes, I'll add a readwrite version of itemID.